worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageMappingCallable.java

```diff
@@ -53,11 +60,11 @@
     private int pastReferenceYear;
     private final Map<Integer, Integer> referenceYears = new HashMap<>();
 
-    public CoverageMappingCallable(FhirVersion version, CoverageMapping coverageMapping, BFDClient bfdClient) {
+    public CoverageMappingCallable(FhirVersion version, CoverageMapping coverageMapping, BFDClient bfdClient, ContractForCoverageDTO contract) {
         this.coverageMapping = coverageMapping;
         this.bfdClient = bfdClient;
         this.completed = new AtomicBoolean(false);
-        this.year = getCorrectedYear(coverageMapping.getContract(), coverageMapping.getPeriod().getYear());
+        this.year = getCorrectedYear(contract, coverageMapping.getPeriod().getYear());
```

Comment:
Move this method to within the ContractForCoverageDTO class.

This constructor should just take a year parameter.  Do all the test corrections outside of this class.

BTW, rename contract to contractDTO when referring to the DTO object.  Otherwise, it makes the code very confusing.